src/main/java/com/movie_collection/gui/controllers/BaseController.java

```diff
@@ -95,35 +95,21 @@ private void setCategoriesScrollPane(List<Category> allCategories) {
 
                     // Setting on the action for switching views
                     categoryBtn.setOnAction(event -> {
-                        Parent parent = null;
-                        try {
-                            parent = loadNodesView(ViewType.MOVIES);
-                        } catch (IOException e) {
-                            throw new RuntimeException(e);
-                        }
+                        Parent parent = tryToLoadView();
```

Comment:
this could also be chucked inside the params but its more so we can actually read it hah